CHANGELOG.md

```diff
@@ -12,6 +11,71 @@ PRs and issues are linked, so you can find more about it. Thanks to [ChangelogLi
+- [#5661] [EarlyReturn] Add ReturnBinaryOrToEarlyReturnRector, Thanks to [@samsonasik]
+- [#5669] [Meta] Add Monorepo Builder, Thanks to [@samsonasik]
+- [#5672] [Naming] Add RenameForeachValueVariableToMatchExprVariableRector, Thanks to [@samsonasik]
+- [#5700] [Rector-CI] Update EndBug/add-and-commit to v7.0.0, Thanks to [@samsonasik]
+- [#5625] [TypeDeclaration] Add types when dataProviders that yield are used, Thanks to [@addshore]
+
+### Changed
+
+- [#5649] [Bin] Handle possible PHPStan Stubs runtime locations, Thanks to [@samsonasik]
+- [#5699] [CodeQuality] Handle Named Argument in ThrowWithPreviousExceptionRector, Thanks to [@samsonasik]
+- [#5691] [CodingStyle] Skip RemoveUnusedAliasRector on used in ClassConstFetch, Thanks to [@samsonasik]
+- [#5694] [DX] Narrow wobly strings to strict ObjectType()  [#2]
+- [#5693] [DX] Narrow wobly strings to strict ObjectType()
+- [#5677] [DeadDocBlock] Skip RemoveUselessParamTagRector on Generic Type, Thanks to [@samsonasik]
+- [#5675] [DeadDocBlock] Skip RemoveUselessReturnTagRector on Generic Type, Thanks to [@samsonasik]
+- [#5711] [EarlyReturn] Handle Return Parent next if on ChangeAndIfToEarlyReturnRector, Thanks to [@samsonasik]
+- [#5706] [EarlyReturn] Handle Always return on ChangeAndIfToEarlyReturnRector, Thanks to [@samsonasik]
+- [#5679] [EarlyReturn] Register ReturnBinaryOrToEarlyReturnRector to early-return config set, Thanks to [@samsonasik]
+- [#5660] [EarlyReturn] Skip ReturnAfterToEarlyOnBreakRector on non-linear previous assignment before foreach, Thanks to [@samsonasik]
+- [#5718] [Naming] Handle UnderscoreToCamelCaseLocalVariableNameRector in used in next and previous assign variable, Thanks to [@samsonasik]
+- [#5682] [Naming] Various Improve RenameForeachValueVariableToMatchExprVariableRector, Thanks to [@samsonasik]
+- [#5686] [NodeTypeResolver] Handle object type or nullable object type from Parameter as Identifier, Thanks to [@samsonasik]
+- [#5671] [Privatization] Skip PrivatizeLocalOnlyMethodRector on use of SymfonyRouteTagValueNode, Thanks to [@samsonasik]
+- [#5692] use getNodeTypes() with class-string generic type
+- [#5665] Make use of static reflection from PHPStan
+- [#5662] update monorepo-builder syntax
+- [#5656] use RequireStringArgumentInConstructorRule
+- [#5655] Bump to Symplify 9.2.1
+- [#5698] [automated] Re-Generate Nodes/Rectors Documentation, Thanks to [@github-actions][bot]
+- [#5657] [static] Improve types
+
+### Fixed
+
+- [#5659] [TypeDeclaration] Fixes for array dataProviders with multiple types, Thanks to [@addshore]
+- [#5695] Fix url to nodes-overview, Thanks to [@staabm]
+
+### Removed
+
+- [#5709] [DX] Removing more parent abstract circular classes
+- [#5717] [Downgrade] Remove excessive abstract class inheritance, use single service
+- [#5714] Removing parent classes
+- [#5713] remove unmatched errors + require string in object types to avoid prefixing of generics objects
+
 ### Added
```

Comment:
There is "added" section twice now, it should be just once per release

Reply:
Yes, that's seems the limitation of current changelog merge

Reply:
The current state of merge seems quite problematic. We have to find out what is the last tag, if it is relased or not and then somehow put merge two lists. It leads mostly to bugs and broken output.

Instead the whole changelog could be generated instead.
The only limite we can add is `--since-tag`, so we can create changelog for specific version, e.g. Symplify 8+.

That would save us lot of content juggling :) 

What do you think?

Reply:
that seems the way to go, is the `since-tag` exists?

Reply:
Not yet, both is just theory

Reply:
Could you handle this in Symplify?

Reply:
I will try